src/pages/CommitDetailPage/subRoute/FilesChangedTab/FilesChangedTab.tsx

```diff
@@ -44,6 +46,9 @@ function FilesChanged() {
 
   return (
     <Suspense fallback={<Loader />}>
+      <div className="flex justify-end bg-ds-gray-primary p-2">
```

Comment:
I think this div will consume space if `ComponentsSelector` returns null. Can we do a quick check that the flag is enabled before rendering these 3 lines? You can use `useFlags` and do something like 
```

{componentsSelect && (
  <div className="">
          ...
  </div>
)}
```

Reply:
👍